cleverhans/utils.py

```diff
@@ -3,12 +3,17 @@
 from __future__ import print_function
 from __future__ import unicode_literals
 
-import os
+from distutils.version import LooseVersion
 import keras
-import matplotlib.pyplot as plt
 from keras.models import Sequential
 from keras.layers import Dense, Dropout, Activation, Flatten
-from keras.layers import Convolution2D
+import matplotlib.pyplot as plt
+import os
+
+if LooseVersion(keras.__version__[0]) >= LooseVersion('2.0.0'):
```

Comment:
What the `[0]`? Shouldn't it work without that?
The `[0]` is what makes it brittle (imagine Keras version 10)